### PR TITLE
Make the Constants Class of src and test abstract.

### PR DESCRIPTION
**Pull request by Incarnation-p-lee**
* As we do not need to create instance for these class.

Signed-off-by: Pan Li <panli@microsoft.com>

### CODE REVIEW

File: src/main/java/com/microsoft/spring/data/gremlin/common/Constants.java [hunk 1]
```diff
@@ -5,7 +5,7 @@
  */
 package com.microsoft.spring.data.gremlin.common;
 
-public class Constants {
+public abstract class Constants {
```
Comment (warrenzhu25) [inline src/main/java/com/microsoft/spring/data/gremlin/common/Constants.java]: I don't see any benefits of making it abstract. What's the motivation behind this?

Comment (Incarnation-p-lee) [reply src/main/java/com/microsoft/spring/data/gremlin/common/Constants.java]: Some static analyzer may complain about the class has not default constructor, So we may need to add one for that. For avoiding this, and this class may not need any instance. Abstract may be a better option here.

Comment (Incarnation-p-lee) [reply src/main/java/com/microsoft/spring/data/gremlin/common/Constants.java]: This is the convention for this situation of most java developers ? If so, just keep original code is OK. 
**PS**: Sorry I am not familiar with this. I think private constructor may looks kind of tricky and misleading. That's why I changed to abstract.

**Comment by Incarnation-p-lee** [comment]
Close this PR for following the java convention.

**Pull request closed without merging**